src/Microsoft.VisualStudio.Editors/PropPages/CSharpLanguageVersion.vb

```diff
@@ -8,6 +8,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
     Friend Class CSharpLanguageVersion
 
         Private Const s_languageVersion_Default As String = "default"
+        Private Const s_languageVersion__DisplayNameForDefault As String = "C# latest major version (default)"
```

Comment:
If you haven't done this yet (run the localizer) - don't worry about it, I'm tracking running the localizer tool just once for this release: https://github.com/dotnet/roslyn-project-system/issues/1794.